test/clj/rems/test_browser.clj

```diff
@@ -2613,24 +2619,98 @@
 
         (is (= ["Delete: Failed"
                 "It is in use by:"
-                "Category: E2E Ancestor category (EN)"]
+                (str "Category: " (btu/context-getx :category-name) " Ancestor")]
                (-> (btu/get-element-text-el (btu/query {:css "#flash-message-top"}))
                    (str/split-lines))))))
 
     (testing "delete category"
       (testing "should contain created categories before delete"
-        (navigate-to-categories)
+        (go-to-categories)
         (is (btu/eventually-visible? :categories))
-        (is (= #{"Edited title (EN)" "E2E Ancestor category (EN)"}
+        (is (= #{(str (btu/context-getx :category-name) " Edited") (str (btu/context-getx :category-name) " Ancestor")}
                (->> (set (slurp-categories-by-title))
-                    (intersection #{"Edited title (EN)" "E2E Ancestor category (EN)"})))))
+                    (intersection #{(str (btu/context-getx :category-name) " Edited") (str (btu/context-getx :category-name) " Ancestor")})))))
+
       (click-row-action [:categories]
-                        {:fn/text "E2E Ancestor category (EN)"}
+                        {:fn/text (str (btu/context-getx :category-name) " Ancestor")}
                         (select-button-by-label "View"))
       (btu/scroll-and-click :delete)
       (btu/wait-has-alert)
       (btu/accept-alert)
+
       (is (btu/eventually-visible? :categories))
-      (is (= #{"Edited title (EN)"}
+      (is (= #{(str (btu/context-getx :category-name) " Edited")}
              (->> (set (slurp-categories-by-title))
-                  (intersection #{"Edited title (EN)" "E2E Ancestor category (EN)"})))))))
+                  (intersection #{(str (btu/context-getx :category-name) " Edited") (str (btu/context-getx :category-name) " Ancestor")})))))))
+
+(deftest test-catalogue-tree
```

Comment:
would you see value in testing with `check-axe` in this test? application tests don't open subcategories and i think they use table rather than catalogue tree.

Reply:
Yeah since it is meant for the applicant it makes sense to have it covered. I'll add a separate task though, because I want to check the state of the check too.